include/api.php

```diff
@@ -2049,6 +2049,9 @@ function api_format_messages($item, $recipient, $sender) {
 				'recipient_screen_name' => $recipient['screen_name'],
 				'sender'                => $sender,
 				'recipient'             => $recipient,
+				'title'			=> "",
```

Comment:
No, title is not new. When you look into master branch in line 1914 you see `$ret['title'] = $item['title'] ;`. This is adding 'title' to the array as it is missing in the array definition. To keep the order (... sender, recipient, then title, then the new fields), I have added the field to the array definition.